chart/prometheus-msteams/README.md

```diff
@@ -86,6 +87,26 @@ helm upgrade --install prometheus-msteams \
 
 Otherwise you can also set the value by specifying the template data directly via values file.
 
+### Customise Messages per MS Teams Channel
+
+To define a custom message template per MS Teams channel you can use the following configuration.
+Like descibed in [Customise messages to MS Teams](#customise-messages-to-ms-teams) you can use the `--set-file` flag
```

Comment:
Please be more concise and provide an example how to use `--set-file` as it is not that easy when working with arrays (see my example in my review comment below).

Reply:
That's true, I'll update this part to be more concise how to use --set-file with an array.

Reply:
That's true, I'll update this part to be more concise how to use --set-file with an array.